sql/src/lib.rs

```diff
@@ -8,18 +8,40 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+extern crate ordered_float;
```

Comment:
It seems like most Rust libraries do all the `extern crate` declarations in `lib.rs`.  This avoids some shenanigans with nested directories and rooted `use` statements.  I noticed you haven't done this in some parts of the `query-*` hierarchy.  I see no guidance at https://aturon.github.io/style/imports.html.

Do you have a strong opinion?  (This particular case does not conflict, just trying to straighten this out.)